src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
That seems reasonable, I didn't realize it was a problem elsewhere, I was just addressing the problem noted in the bug.